reports-scheduler/src/main/kotlin/org/opensearch/reportsscheduler/action/DeleteReportDefinitionAction.kt

```diff
@@ -51,7 +51,7 @@ internal class DeleteReportDefinitionAction @Inject constructor(
     actionFilters,
     ::DeleteReportDefinitionRequest) {
     companion object {
-        private const val NAME = "cluster:admin/opendistro/reports/definition/delete"
+        private const val NAME = "cluster:admin/opensearch/reports/definition/delete"
```

Comment:
Seems these should not be changed as security plugin didn't change them yet. ref: https://github.com/opensearch-project/security/issues/1178

Reply:
reverting